concent_api/concent_api/tests/test_storage_cluster_address_check.py

```diff
@@ -0,0 +1,74 @@
+from django.test                import override_settings
+from django.conf                import settings
+from django.test                import TestCase
+
+from concent_api.system_check   import create_error_36_storage_cluster_address_does_not_end_with_slash
+from concent_api.system_check   import create_error_38_storage_cluster_address_is_not_valid_url
+from concent_api.system_check   import create_error_39_storage_server_internal_address_is_not_set
+from concent_api.system_check   import check_settings_storage_cluster_address
```

Comment:
according to our coding style it should be without additional space, just:
```
from django.test import override_settings
from django.conf import settings
from django.test import TestCase

from concent_api.system_check import create_error_36_storage_cluster_address_does_not_end_with_slash
from concent_api.system_check import create_error_38_storage_cluster_address_is_not_valid_url
...
```

Reply:
:+1: done